registry/server/common/services/preProcessResponse.ts

```diff
@@ -1,25 +1,23 @@
 import _ from 'lodash/fp';
```

Comment:
Why do you need to rename all these vars which unnecessarily cause a lot of side changes?

Reply:
Removed